tests/commands/test_changelog_command.py

```diff
@@ -234,3 +236,26 @@ def test_changlog_incremental_keep_a_changelog_sample(mocker, capsys):
         out
         == """# Changelog\nAll notable changes to this project will be documented in this file.\n\nThe format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),\nand this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).\n\n\n## Unreleased \n\n### Feat\n\n- add more stuff\n- add new output\n\n### Fix\n\n- mama gotta work\n- output glitch\n\n## [1.0.0] - 2017-06-20\n### Added\n- New visual identity by [@tylerfortune8](https://github.com/tylerfortune8).\n- Version navigation.\n\n### Changed\n- Start using "changelog" over "change log" since it\'s the common usage.\n\n### Removed\n- Section about "changelog" vs "CHANGELOG".\n\n## [0.3.0] - 2015-12-03\n### Added\n- RU translation from [@aishek](https://github.com/aishek).\n"""
     )
+
+
+@pytest.mark.usefixtures("tmp_commitizen_project")
+def test_changlog_hook(mocker, config):
+    hook_mock = mocker.Mock()
+
+    def changelog_hook(full_changelog: str, partial_changelog: Optional[str]) -> None:
+        if partial_changelog:
+            hook_mock()
+        if full_changelog:
+            hook_mock()
```

Comment:
How about sending `full_changelog` and `partial_changelog` into `hook_mock` and use `assert_called_with`?